### ISSUE TITLE
Thread panicked by request from browser

### ISSUE THREAD

**Issue by mehmetdavut** [oakserver/oak#213]
I've just tried a basic api with oak http server on deno. Everything works well when I call api endpoints from Postman, there is no any error. But when I try to request api endpoints from a browser, I get an error like this and server stops suddenly. How can I fix this?

```
web_1       | thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: FromHexError(OddLength)', /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/bson-0.14.1/src/bson.rs:575:39
web_1       | stack backtrace:
web_1       |    0:     0x7fbf076443d4 - backtrace::backtrace::libunwind::trace::hc1c4a1d8ad423b97
web_1       |                                at /cargo/registry/src/github.com-1ecc6299db9ec823/backtrace-0.3.46/src/backtrace/libunwind.rs:86
web_1       |    1:     0x7fbf076443d4 - backtrace::backtrace::trace_unsynchronized::h82274781060cb056
web_1       |                                at /cargo/registry/src/github.com-1ecc6299db9ec823/backtrace-0.3.46/src/backtrace/mod.rs:66
web_1       |    2:     0x7fbf076443d4 - std::sys_common::backtrace::_print_fmt::h2a45d89b653a4da8
web_1       |                                at src/libstd/sys_common/backtrace.rs:78
web_1       |    3:     0x7fbf076443d4 - <std::sys_common::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt::h41a0a93ab85e6aa1
web_1       |                                at src/libstd/sys_common/backtrace.rs:59
web_1       |    4:     0x7fbf0766ebfc - core::fmt::write::hdaea18585065a96d
web_1       |                                at src/libcore/fmt/mod.rs:1069
web_1       |    5:     0x7fbf0763d643 - std::io::Write::write_fmt::h0cea70c809005252
web_1       |                                at src/libstd/io/mod.rs:1504
web_1       |    6:     0x7fbf07646a65 - std::sys_common::backtrace::_print::hd95f9978cc145ca4
web_1       |                                at src/libstd/sys_common/backtrace.rs:62
web_1       |    7:     0x7fbf07646a65 - std::sys_common::backtrace::print::hfb25ca2291be47d0
web_1       |                                at src/libstd/sys_common/backtrace.rs:49
web_1       |    8:     0x7fbf07646a65 - std::panicking::default_hook::{{closure}}::h44f76cee5dc8591c
web_1       |                                at src/libstd/panicking.rs:198
web_1       |    9:     0x7fbf076467a2 - std::panicking::default_hook::h198e1a712910f1e6
web_1       |                                at src/libstd/panicking.rs:218
web_1       |   10:     0x7fbf076470c2 - std::panicking::rust_panic_with_hook::hc0b4730bb8013f9d
web_1       |                                at src/libstd/panicking.rs:511
web_1       |   11:     0x7fbf07646cab - rust_begin_unwind
web_1       |                                at src/libstd/panicking.rs:419
web_1       |   12:     0x7fbf0766d371 - core::panicking::panic_fmt::h1ac71ad045d55416
web_1       |                                at src/libcore/panicking.rs:111
web_1       |   13:     0x7fbf0766d193 - core::option::expect_none_failed::hb40004d71d61b685
web_1       |                                at src/libcore/option.rs:1268
web_1       |   14:     0x7fbf075e1d32 - bson::bson::Bson::from_extended_document::h41bab5ef8b085c90
web_1       |   15:     0x7fbf075df66e - <bson::bson::Bson as core::convert::From<serde_json::value::Value>>::from::hacad07e9a30fb265
web_1       |   16:     0x7fbf075eb09d - <bson::ordered::OrderedDocument as core::iter::traits::collect::FromIterator<(alloc::string::String,bson::bson::Bson)>>::from_iter::hd187b5d421730902
web_1       |   17:     0x7fbf075df662 - <bson::bson::Bson as core::convert::From<serde_json::value::Value>>::from::hacad07e9a30fb265
web_1       |   18:     0x7fbf073151f9 - <core::future::from_generator::GenFuture<T> as core::future::future::Future>::poll::h11b1f6e237414c3b
web_1       |   19:     0x7fbf072dc8ba - <futures_util::future::future::map::Map<Fut,F> as core::future::future::Future>::poll::h511a2e56f8a67bbc
web_1       |   20:     0x7fbf072ddc77 - <futures_util::future::future::flatten::Flatten<Fut,<Fut as core::future::future::Future>::Output> as core::future::future::Future>::poll::h1d921dcc3653d0f9
web_1       |   21:     0x55590adaa7e4 - <futures_util::future::future::map::Map<Fut,F> as core::future::future::Future>::poll::ha8769a3d272a54f8
web_1       |   22:     0x55590adb8089 - <futures_util::future::future::Map<Fut,F> as core::future::future::Future>::poll::he9cd49154b6a0a46
web_1       |   23:     0x55590adb3681 - futures_util::stream::stream::StreamExt::poll_next_unpin::h028336e60c227d72
web_1       |   24:     0x55590ad91719 - <deno_core::core_isolate::CoreIsolate as core::future::future::Future>::poll::h85eb3695ef083564
web_1       |   25:     0x55590adb174f - <deno_core::es_isolate::EsIsolate as core::future::future::Future>::poll::ha058ce5e1d8a299e
web_1       |   26:     0x55590a35d4a5 - <core::future::from_generator::GenFuture<T> as core::future::future::Future>::poll::h825a6c9da218875d
web_1       |   27:     0x55590a2d5370 - std::thread::local::LocalKey<T>::with::h33786aa74c13d92e
web_1       |   28:     0x55590a4283a7 - tokio::macros::scoped_tls::ScopedKey<T>::set::h00d04386b4f206d2
web_1       |   29:     0x55590a46815d - tokio::runtime::basic_scheduler::BasicScheduler<P>::block_on::ha931fafdb3df367a
web_1       |   30:     0x55590a429e4d - tokio::runtime::context::enter::hbdf551f3a7afc2ad
web_1       |   31:     0x55590a4dbfc4 - deno::tokio_util::run_basic::h07af1fb9e10205f6
web_1       |   32:     0x55590a449ed3 - deno::main::h5ab1a96ddc066c41
web_1       |   33:     0x55590a2d26b3 - std::rt::lang_start::{{closure}}::h65be8e161a2027c2
web_1       |   34:     0x55590b9521d8 - std::rt::lang_start_internal::{{closure}}::h7a212202ff76034b
web_1       |                                at src/libstd/rt.rs:52
web_1       |   35:     0x55590b9521d8 - std::panicking::try::do_call::h6d214a73427d759b
web_1       |                                at src/libstd/panicking.rs:331
web_1       |   36:     0x55590b9521d8 - std::panicking::try::hc078f0e11721d1cb
web_1       |                                at src/libstd/panicking.rs:274
web_1       |   37:     0x55590b9521d8 - std::panic::catch_unwind::hb31c05be30625612
web_1       |                                at src/libstd/panic.rs:394
web_1       |   38:     0x55590b9521d8 - std::rt::lang_start_internal::hcf7fb98a775d5af0
web_1       |                                at src/libstd/rt.rs:51
web_1       |   39:     0x55590a44c242 - main
web_1       |   40:     0x7fbf088210b3 - __libc_start_main
web_1       |   41:     0x55590a11e7b9 - _start
web_1       |   42:                0x0 - <unknown>
```

**Comment by kitsonk** [comment]
It is unlikely to be an oak issue, since it is buried inside Deno, but would need more information to be able to try to reproduce, like `deno --version` and a minimal example of code you are running.

**Comment by mehmetdavut** [comment]
@kitsonk I think so, but since the problem starts with browser request, I thought that It would be better to start right here. So, now I've tried it and it gave me same error. Then I've deleted some lines to make my code more minimalistic, It started to work as I expected. Then I've pasted old codes which gave me error at the beginning, now It works as I expected again without any error.

So I think it is not reproducable for now and I wish to close this issue. And If I can find a way to reproduce the error, I will re-open the issue.

My deno versions is:

```
$ deno --version
deno 1.2.0
v8 8.5.216
typescript 3.9.2
```
